openassessment/templates/openassessmentblock/oa_grade_available_responses.html

```diff
@@ -0,0 +1,32 @@
+{% load i18n %}
+{% spaceless %}
+<div class="wrapper wrapper--xblock wrapper--openassessment theme--basic">
+    <div class="openassessment problem">
+        <div class="wrapper--grid">
+            {% if title %}
+                <h3 class="openassessment__title problem__header">{% trans title %}</h3>
+            {% endif %}
+
+            {% if staff_assessment_required %}
+            <div class="openassessment__staff-area">
+                <div class="openassessment__staff-grading wrapper--staff-grading">
+                    <div class="staff-grading ui-staff">
+                        <h2 class="staff-grading__title">
```

Comment:
Is this h2 below the h3?  Please review the order of the headers in this file.  This order is important for accessibility.